.github/workflows/daily-e2e.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Daily E2E Tests'
+
+on:
+   pull_request:
```

Comment:
This is needed only for testing inside this PR. I'll remove the line before merging.